sktime/forecasting/base.py

```diff
@@ -30,15 +34,19 @@ def fit(self, y_train, fh=None, X_train=None):
         """Fit model to training data"""
         raise NotImplementedError
 
-    def predict(self, fh=None, X=None, return_conf_int=False, alpha=0.05):
+    def predict(self, fh=None, X=None, return_conf_int=False, alpha=_DEFAULT_ALPHA):
```

Comment:
Good to define this only in one place, but how will this be displayed in the online documentation? Perhaps use `None` as default and then assign `_DEFAULT_ALPHA` inside method when `None` is passed?

Reply:
It will get resolved to the actual value in the documentation - so it would be displayed as `0.05`, which is arguably better than defaulting to `None` - well, from a documentation perspective at least.

One thought I had though was that we could maybe reduce the number of parameters needed here. `alpha=None` could be interpreted as "don't calculate intervals", removing the need for `return_conf_int`. For this to work though, we would need to default to ignoring the user request when intervals are requested for a forecaster  that doesn't have `pred_errs` implemented (rather than raising an explicit error).

Which approach do you think is best?

Reply:
* Okay, makes sense to keep `_DEFAULT_ALPHA`
* I'd keep it explicit with the `return_conf_int` option.